src/transformers/pipelines/__init__.py

```diff
@@ -302,6 +305,18 @@
         "default": {"model": {"pt": ("facebook/detr-resnet-50-panoptic", "fc15262")}},
         "type": "image",
     },
+    "image2text-generation": {
+        "impl": Image2TextGenerationPipeline,
+        "tf": (TFAutoModelForVision2Seq,) if is_tf_available() else (),
+        "pt": (AutoModelForVision2Seq,) if is_torch_available() else (),
+        "default": {
+            "model": {
+                "pt": ("nlpconnect/vit-gpt2-image-captioning", "27b41be"),
+                "tf": ("ydshieh/vit-gpt2-coco-en", "65636df"),
```

Comment:
Why are the two models different ?
 
 Which one is better @ydshieh do you have an advice here maybe ?
 
 Whenever possible we put the best default model here, usually it's the same for both TF and PT, but it depends of implementations.

Reply:
`"nlpconnect/vit-gpt2-image-captioning"` mentions 

> This is an image captioning model training by @ydshieh in flax, this is pytorch version of https://huggingface.co/ydshieh/vit-gpt2-coco-en-ckpts model.

So I believe it's just the same checkpoint. In `ydshieh/vit-gpt2-coco-en` there is also PyTorch checkpoint too, so we can use the same repo.

Reply:
Should I just use `ydshieh/vit-gpt2-coco-en` as a default for both PyTorch and Tensorflow then? Or should we add the tf weights to `nlpconnect/vit-gpt2-image-captioning` which seems more popular.

Reply:
@ydshieh,
It seems that the `preprocessor_config.json` in `ydshieh/vit-gpt2-coco-en` is missing an entry. I opened a PR [here](https://huggingface.co/ydshieh/vit-gpt2-coco-en/discussions/1) to fix it.